tidy3d/plugins/autograd/invdes/filters.py

```diff
@@ -0,0 +1,66 @@
+from typing import Tuple, Union
+from functools import partial
+
+import numpy as np
+
+from ..functions import convolve
+from ..types import KernelType, PaddingType
+from ..utilities import make_kernel
+
+
+def make_filter(
+    filter_type: KernelType,
+    size: Union[int, Tuple[int, ...]],
+    *,
```

Comment:
what is this?

Reply:
I bug people a lot to use keyword arguments as much as possible. Didn't know you could enforce it like this, interesting. 🤔